### PR TITLE
Add FFQueue version 1.7.52

### PR DESCRIPTION
**Pull request by MCOfficer**

### CODE REVIEW

File: bucket/ffqueue.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "description": "(Yet another) graphical user interface for FFMpeg with comprehensive support for both the basic features but also the more advanced features like filtergraphs.",
+    "homepage": "http://ffqueue.bruchhaus.dk/",
+    "version": "1.7.52",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://gist.githubusercontent.com/MCOfficer/f308c0b562272fa5b4c6bb3163c72509/raw/dd293a390bfe950a032c2547f33c462638b06a51/ffqueue-1.2-license.txt"
```
[the remark this answers is not in the document]
Comment (MCOfficer) [reply bucket/ffqueue.json]: Did you find one?

[the remark this answers is not in the document]
Comment (Ash258) [reply bucket/ffqueue.json]: You had to copy that file from somewhere. Where did you find that license file? Or you wrote it yourself?

[the remark this answers is not in the document]
Comment (Ash258) [reply bucket/ffqueue.json]: Source code is under GPL-3.0 and binaries are under that one mentioned here: https://ffqueue.bruchhaus.dk/Download.aspx.
I would use link to that download page.

> Two licenses for one program? What?!??
> I am going to continue to release the build on this website under the simple and light weight FFQueue license. The source code, however, is released under the GPL3 license in order to protect the code of derived works from closure in the future.

**Comment by r15ch13** [comment]
A default `ffqueue.cfg` could be created containing the preconfigured path to `ffmpeg` (if `depends` is set)

```config
FFQUEUE_CONFIG_VERSION_10
ffmpeg_path=D:\scoop\apps\ffmpeg\current\bin\ffmpeg.exe
```

**Comment by MCOfficer** [comment]
while considering ffmpeg it occured to me that we also have at least 1 alternative ffmpeg (the nightly). can i list dependencies so only one of them has to be installed? i know suggests has something like that, in the form of features.

i'm also not happy with using gists for licenses, do you have any other suggestions? Note that i did the same thing in #2050

File: bucket/ffqueue.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (MCOfficer) [reply bucket/ffqueue.json]: i hand't considered that, but in hindsight it's very reasonable.

**Comment by r15ch13** [comment]
Add a suggestion to install `ffmpeg`/`ffmpeg-nightly`/`ffmpeg-shared`

**Pull request merged**